src/abstractdataframe/iteration.jl

```diff
@@ -20,6 +20,14 @@ end
 Base.summary(dfrs::DataFrameRows) = "$(length(dfrs))-element DataFrameRows"
 Base.summary(io::IO, dfrs::DataFrameRows) = print(io, summary(dfrs))
 
+# Trying to iterate over a DataFrame directly (for row in df) tends to
```

Comment:
Can you remove this comment? People can easily understand why this method has been defined, and if not they can look at the git history.

Reply:
Done.